src/jit/codegen.h

```diff
@@ -686,12 +686,18 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 public:
     void psiBegProlog();
 
+    // When ESP changes, all scopes relative to ESP have to be updated.
```

Comment:
Sorry, I understood that the idea of the comments on header file were to be the same but in less than 3 lines to avoid going and search the function definition. I will remove it.

Reply:
I have removed the comments that are exactly the same. Thanks!